packages/client/validation/makeAgendaItemSchema.ts

```diff
@@ -1,10 +1,18 @@
-import {compositeId, id} from './templates'
 import legitify from './legitify'
 import Legitity from './Legitity'
+import {compositeId, id} from './templates'
+
+export const AGENDA_ITEM_MAX_CHARS = 63
+
+export const content = (value: Legitity) =>
```

Comment:
I'm not sure about naming here...

I think it's better to keep the validation logic for some model in one module (high coupling). This way we avoid  inconsistent logic changes associated with this model.

So I keep it there instead of moving to `packages/client/validation/templates.ts` or similar.

Reply:
+1 `contentValidator` might be a bit better